gulp/tasks/test_debug.js

```diff
@@ -0,0 +1,26 @@
+(function () {
+    'use strict';
+
+    var gulp = require('gulp'),
+        karma = require('karma').server,
+        path = require('path'),
+        paths = {
+            spec: [
+                'test/specs/**/*.js'
```

Comment:
@talbs Thanks so much for the great links! I haven't really dug into node before, so this is really helpful. I'm going to adopt this technique in all the configuration files.